### PR TITLE
Add split and piggy-bank-check icons

### PR DESCRIPTION
**Pull request by dcmouyard**
Add split and piggy-bank-check icons with round variants.

## Additions

- piggy-bank-check.svg
- piggy-bank-check-round.svg
- split.svg
- split-round.svg

new-icons

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Internet Explorer 8, 9, 10, and 11
- [ ] Edge
- [ ] iOS Safari
- [ ] Chrome for Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

**Comment by jimmynotjim** [comment]
@dcmouyard could you add the icons to the appropriate section in https://github.com/cfpb/capital-framework/blob/canary/src/cf-icons/usage.md so they'll be displayed in the docs site. A few screenshots in the opening comment would probably help too.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
--
+- **cf-icons:** Add piggy-bank-check and split icons
```
Comment (jimmynotjim) [inline CHANGELOG.md]: You'll need to format this for the bot to ingest it as:

```
**cf-icons:** [MINOR] Add piggy-bank and split icons
```

Comment (anselmbradford) [reply CHANGELOG.md]: Do we document this syntax anywhere?

Comment (jimmynotjim) [reply CHANGELOG.md]: In the contributing doc https://github.com/cfpb/capital-framework/blob/canary/CONTRIBUTING.md#changing-the-codebase

Comment (anselmbradford) [reply CHANGELOG.md]: Hrm, except the example linked to there [is wrong](https://github.com/cfpb/capital-framework/pull/291/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR12) 😬

Comment (jimmynotjim) [reply CHANGELOG.md]: Oh weird, it must have changed at one point. Want to update that to point to a more recent PR?

Comment (anselmbradford) [reply CHANGELOG.md]: Done! Merged it straight to `canary` 🔥 .

**Comment by dcmouyard** [comment]
@jimmynotjim I added a screenshot of the SVGs, added the icons to the usage documentation, and updated changelog.

**Comment by dcmouyard** [comment]
I just noticed that these new icons have empty space on the sides, unlike the existing icons. I’ll get new icons from the designer.

**Comment by dcmouyard** [comment]
The icons were updated to remove the extra horizontal space.

**Comment by dcmouyard** [comment]
@jimmynotjim @Scotchester Can someone restart the build test?

**Comment by Scotchester** [comment]
Done.

**Comment by jimmynotjim** [comment]
👋 

Looks like the NPM token is bad cc @contolini

**Comment by anselmbradford** [comment]
Can this be merged?

**Comment by anselmbradford** [comment]
Merging from chat discussion.

**Pull request merged**